cmd/install.go

```diff
@@ -60,23 +51,27 @@ func installCommandAction(cmd *cobra.Command, _ []string) error {
 	if err != nil {
 		return errors.Wrap(err, "could not create kibana client")
 	}
-	kibanaVersion, err := kibanaClient.Version()
-	if err != nil {
-		return err
-	}
-	v, err := semver.NewVersion(kibanaVersion.Number)
-	if err != nil {
-		return fmt.Errorf("invalid Kibana version")
-	}
 
-	installer, err := newInstaller(zipPathFile, packageRootPath, v, skipValidation)
-	if err != nil {
-		return err
+	if packageRootPath == "" {
+		var found bool
+		var err error
+		packageRootPath, found, err = packages.FindPackageRoot()
+		if !found {
+			return errors.New("package root not found")
+		}
+		if err != nil {
+			return errors.Wrap(err, "locating package root failed")
+		}
 	}
```

Comment:
As it is now the tests, if it is used the zip parameter this could be executed outside of the package folder:
- https://github.com/elastic/elastic-package/blob/main/scripts/test-install-zip.sh#L119

Because of that, it is possible that it is not found the package root.

Should it be added another test to install zip directly from the package folder?

Reply:
I see that you already fixed this :+1: